### DISCUSSION TITLE
SSH Credentials

### DISCUSSION THREAD

**Discussion by dcveteran** [OpenVoiceOS/ovos-buildroot#274] [Q&A] [answered]
I'm working with a buildroot image from Jan 2023 today, anyone know the default SSH credentials? I have tried many variations of ovos, pi, root, and admin with no luck. It is definitely not ovos, ovos as listed in the documentation (must have change with newer version).

Thanks!

**Comment by j1nx** [comment, marked as answer]
That older version probably use;

mycroft/mycroft

otherwise it is;

ovos/ovos

**Reply by dcveteran** [reply]
Perfect, thanks!

**Reply by j1nx** [reply]
Any change you still have the image file at hand and be able to upoad it for us? (Lost older image files)

**Reply by dcveteran** [reply]
Yes, where would you like it? I have found this one to be the most stable for my setup with the GUI.

**Reply by j1nx** [reply]
What ever upload system suits you best. I will download it and let you know when done so you can remove it again.

If the server ends up to be a private space, it might be best to send that over email to dev@openvoiceos.org

Thank you very much and indeed, we see that version for the buildroot system as the "stable" <0.0.8 ovos version.

**Reply by dcveteran** [reply]
@j1nx, here you go:
https://www.dropbox.com/scl/fi/wqmzsrglc993n0zcvg7f7/OpenVoiceOS_rpi4_64-gui_dev-202301171328.img.zip?rlkey=vb9u3ungaots4mm1k7pw88mkj&st=ikp5d4go&dl=0

**Reply by j1nx** [reply]
Thank you very much. Downloaded the file, so if you want to delete it, you can.
